pandas/tests/indexes/datetimes/test_ops.py

```diff
@@ -540,6 +540,16 @@ def test_shift(self):
             shifted = rng.shift(1, freq=CDay())
             assert shifted[0] == rng[0] + CDay()
 
+    def test_shift_periods(self):
+        # GH #22458 : argument 'n' was deprecated in favor of 'periods'
+        idx = pd.DatetimeIndex(start=START, end=END,
+                               periods=3)
+        tm.assert_index_equal(idx.shift(periods=0), idx)
+        tm.assert_index_equal(idx.shift(0), idx)
+        with tm.assert_produces_warning(FutureWarning,
+                                        check_stacklevel=False):
```

Comment:
Could you change this to true? If the test fails, you may need to increase the stacklevel 1.

Reply:
@TomAugspurger done, test passed too.